modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -160,5 +160,19 @@ print(response.json())
 
 === Token expiration
 
-If you send a request to the Aura API while authenticated with an expired access token, you will receive a 403 Forbidden response.
-You will need to obtain a new token to continue using the API.
+Tokens are temporary access tokens used for authentication when making API requests. 
+They expire after one hour. 
+If you send a request to the Aura API using an expired token, you will receive a 403 Forbidden response. 
+To continue using the API, you must obtain a new token.
+
+In the *Account Details* section of the Aura console, Aura API credentials refer to OAuth clients (Client ID and Client Secret). 
+These credentials do not expire unless deleted and are used to generate access tokens, which do expire.
+
+When an access token is created using a Client ID and Client Secret, the response payload specifies its expiration time in seconds. 
+Once the token expires, the application must request a new one.
+
+
```

Comment:
I think this whole paragraph is redundant, since this is mentioned in the previous section. Maybe add a description of the `expires_in` field in the "Response body example" section after the description of `access_token`?